README.md

```diff
@@ -184,6 +184,12 @@ The following environment variables configure Livebook:
     Enabled by default unless `LIVEBOOK_PASSWORD` is set. Set it to "false" to
     disable it.
 
+  * LIVEBOOK_FORCE_SSL - force SSL connections and enable HSTS.
+    Ensuring no data is ever sent via http, always redirect to https.
+
```

Comment:
Sorry, maybe there was some confusion. We don't need LIVEBOOK_FORCE_SSL anymore, only LIVEBOOK_FORCE_SSL_HOST. :)

Reply:
I've solved it.